testing/scenario_app/android/app/src/main/AndroidManifest.xml

```diff
@@ -33,11 +33,6 @@
             android:launchMode="singleTop"
             android:windowSoftInputMode="adjustResize"
             android:theme="@style/FullScreenScreenshot">
-            <intent-filter>
```

Comment:
Firebase testlab doesn't want us to have multiple activities with this intent filter.